Refresh the vmware2 vm doc (#2860)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/virtualization-vmware2-vm.md

> ## Overview
> 

Reviewer: ```suggestion
```

| --change-short-output --change-long-output | Modify the short/long output that is returned by the plugin. Syntax: --change-short-output=pattern~replacement~modifier Most commonly used modifiers are i (case insensitive) and g (replace all occurrences). Eg: adding --change-short-output='OK~Up~gi' will replace all occurrences of 'OK', 'ok', 'Ok' or 'oK' with 'Up'                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

> | --change-exit                              | Replace an exit code with one of your choice. Eg: adding --change-exit=unknown=critical will result in a CRITICAL state instead of an UNKNOWN state.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| --change-exit                              | Replace an exit code with one of your choice. Example: adding --change-exit=unknown=critical will result in a CRITICAL state instead of an UNKNOWN state.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

| --sampling-period                          | Choose the sampling period (can change the default sampling for counters). Should be not different than 300 or 20.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |

> | --time-shift                               | Can shift the time. We the following option you can average X counters values (default: 0).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |

Reviewer 2: ```suggestion
| --time-shift                               | Can shift the time. With the following option you can average X counters values (default: 0).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

> | --case-insensitive                         | Searchs are case insensitive.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |

Reviewer 2: ```suggestion
| --case-insensitive                         | Searches are case insensitive.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

| --vm-hostname        | VM hostname to check. If not set, we check all VMs.                                                                                                                                                                                |
| --filter             | VM hostname is a regexp. Exemple : --vm-hostname='^((VM-PROD-*))' --filter                                                                                                                                                         |
| --filter-description | Filter also virtual machines description (can be a regexp).                                                                                                                                                                        |
| --filter-os          | Filter also virtual machines OS name (can be a regexp).                                                                                                                                                                            |

> | --scope-datacenter   | Search in following datacenter(s) (can be a regexp).                                                                                                                                                                               |

Reviewer 2: ```suggestion
| --scope-datacenter   | Search in the following datacenter(s) (can be a regexp).                                                                                                                                                                               |
```

> | --scope-cluster      | Search in following cluster(s) (can be a regexp).                                                                                                                                                                                  |

Reviewer 2: ```suggestion
| --scope-cluster      | Search in the following cluster(s) (can be a regexp).                                                                                                                                                                                  |
```

| --critical-*         | Critical threshold. Can be: 'total-cpu', 'total-cpu-mhz', 'cpu-ready', 'cpu'.                                                                                                                                                      |

</TabItem>
<TabItem value="Vm-Datastores-Iops" label="Vm-Datastores-Iops">

| Option                   | Description                                                                                                                                                                                                                        |
|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --vm-hostname            | VM hostname to check. If not set, we check all VMs.                                                                                                                                                                                |

> | --filter                 | VM hostname is a regexp.                                                                                                                                                                                                           |

Reviewer 2: ?

| --filter-description     | Filter also virtual machines description (can be a regexp).                                                                                                                                                                        |
| --filter-os              | Filter also virtual machines OS name (can be a regexp).                                                                                                                                                                            |
| --scope-datacenter       | Search in following datacenter(s) (can be a regexp).                                                                                                                                                                               |
| --scope-cluster          | Search in following cluster(s) (can be a regexp).                                                                                                                                                                                  |

> | --scope-host             | Search in following host(s) (can be a regexp).                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| --scope-host             | Search in the following host(s) (can be a regexp).                                                                                                                                                                                     |
```

| --critical-status        | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{connection\_state}, %{power\_state}                                                                             |
| --warning-* --critical-* | Thresholds. Can be: 'max-total-latency', 'read', 'write'.                                                                                                                                                                          |

</TabItem>
<TabItem value="Vm-Device" label="Vm-Device">

| Option                | Description                                                                                                                                                                  |
|:----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --vm-hostname         | VM hostname to check. If not set, we check all VMs.                                                                                                                          |

> | --filter              | VM hostname is a regexp.                                                                                                                                                     |

Reviewer 2: ?

| --filter-description  | Filter also virtual machines description (can be a regexp).                                                                                                                  |
| --filter-os           | Filter also virtual machines OS name (can be a regexp).                                                                                                                      |
| --scope-datacenter    | Search in following datacenter(s) (can be a regexp).                                                                                                                         |
| --scope-cluster       | Search in following cluster(s) (can be a regexp).                                                                                                                            |

> | --scope-host          | Search in following host(s) (can be a regexp).                                                                                                                               |

Reviewer 2: ```suggestion
| --scope-host          | Search in the following host(s) (can be a regexp).                                                                                                                               |
```

> | --display-description | Display virtual machine description.                                                                                                                                         |

Reviewer 2: ```suggestion
| --display-description | Display the description of the virtual machine.                                                                                                                                         |
```

| --critical-*          | Critical threshold. Can be: 'total-device-connected', 'device-connected'.                                                                                                    |

</TabItem>
<TabItem value="Vm-Limit" label="Vm-Limit">

| Option                   | Description                                                                                                                                                                                                                   |
|:-------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --vm-hostname            | VM hostname to check. If not set, we check all VMs.                                                                                                                                                                           |

> | --filter                 | VM hostname is a regexp.                                                                                                                                                                                                      |

Reviewer 2: ?

| --filter-description     | Filter also virtual machines description (can be a regexp).                                                                                                                                                                   |
| --filter-os              | Filter also virtual machines OS name (can be a regexp).                                                                                                                                                                       |

> | --display-description    | Display virtual machine description.                                                                                                                                                                                          |

Reviewer 2: ```suggestion
| --display-description    | Display the description of the virtual machine.                                                                                                                                                                                          |
```

| --critical-memory-status | Define the conditions to match for the status to be CRITICAL (Default: '%{connection\_state} !~ /^connected$/i \|\| %{limit} != -1'). You can use the following variables: %{connection\_state}, %{power\_state}, %{limit}    |

</TabItem>
<TabItem value="Vm-Memory" label="Vm-Memory">

| Option                | Description                                                                                                                                                                                                                        |
|:----------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --vm-hostname         | VM hostname to check. If not set, we check all VMs.                                                                                                                                                                                |

> | --filter              | VM hostname is a regexp.                                                                                                                                                                                                           |

Reviewer 2: ?

| --filter-description  | Filter also virtual machines description (can be a regexp).                                                                                                                                                                        |
| --filter-os           | Filter also virtual machines OS name (can be a regexp).                                                                                                                                                                            |

> | --scope-datacenter    | Search in following datacenter(s) (can be a regexp).                                                                                                                                                                               |

Reviewer 2: ```suggestion
| --scope-datacenter    | Search in the following datacenter(s) (can be a regexp).                                                                                                                                                                               |
```

| --scope-cluster       | Search in following cluster(s) (can be a regexp).                                                                                                                                                                                  |

> | --scope-host          | Search in following host(s) (can be a regexp).                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| --scope-host          | Search in the following host(s) (can be a regexp).                                                                                                                                                                                     |
```

| --unknown-status      | Define the conditions to match for the status to be UNKNOWN (Default: '%{connection\_state} !~ /^connected$/i or %{power\_state} !~ /^poweredOn$/i'). You can use the following variables: %{connection\_state}, %{power\_state}   |
| --warning-status      | Define the conditions to match for the status to be WARNING (Default: ''). You can use the following variables: %{connection\_state}, %{power\_state}                                                                              |
| --critical-status     | Define the conditions to match for the status to be CRITICAL (Default: ''). You can use the following variables: %{connection\_state}, %{power\_state}                                                                             |
| --units               | Units of thresholds (Default: '%') ('%', 'B').                                                                                                                                                                                     |

> | --free                | Thresholds are on free space left.                                                                                                                                                                                                 |

Reviewer 2: ```suggestion
| --free                | Thresholds are applied on free space left.                                                                                                                                                                                                 |
```

|:----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --vm-hostname         | VM hostname to check. If not set, we check all VMs.                                                                                                           |
| --filter              | VM hostname is a regexp.                                                                                                                                      |
| --filter-description  | Filter also virtual machines description (can be a regexp).                                                                                                   |
| --filter-os           | Filter also virtual machines OS name (can be a regexp).                                                                                                       |
| --scope-datacenter    | Search in following datacenter(s) (can be a regexp).                                                                                                          |

> | --scope-cluster       | Search in following cluster(s) (can be a regexp).                                                                                                             |

Reviewer 2: ```suggestion
| --scope-cluster       | Search in the following cluster(s) (can be a regexp).                                                                                                             |
```

| --scope-host          | Search in following host(s) (can be a regexp).                                                                                                                |
| --display-description | Display virtual machine description.                                                                                                                          |
| --check-consolidation | Check if VM needs consolidation (since vsphere 5.0).                                                                                                          |
| --disconnect-status   | Status if VM disconnected (default: 'unknown').                                                                                                               |

> | --nopoweredon-skip    | Skip check if VM is not poweredOn.                                                                                                                            |

Reviewer 2: ```suggestion
| --nopoweredon-skip    | Skip check if VM is not powered on.                                                                                                                            |
```

| --vm-hostname             | VM hostname to check. If not set, we check all VMs.                                                                                                                                           |
| --filter                  | VM hostname is a regexp.                                                                                                                                                                      |
| --filter-description      | Filter also virtual machines description (can be a regexp).                                                                                                                                   |
| --filter-os               | Filter also virtual machines OS name (can be a regexp).                                                                                                                                       |
| --scope-datacenter        | Search in following datacenter(s) (can be a regexp).                                                                                                                                          |

> | --scope-cluster           | Search in following cluster(s) (can be a regexp).                                                                                                                                             |

Reviewer 2: ```suggestion
| --scope-cluster           | Search in the following cluster(s) (can be a regexp).                                                                                                                                             |
```

| --vm-hostname         | VM hostname to check. If not set, we check all VMs.                                                                                                                                                                                |
| --filter              | VM hostname is a regexp.                                                                                                                                                                                                           |
| --filter-description  | Filter also virtual machines description (can be a regexp).                                                                                                                                                                        |
| --filter-os           | Filter also virtual machines OS name (can be a regexp).                                                                                                                                                                            |

> | --scope-datacenter    | Search in following datacenter(s) (can be a regexp).                                                                                                                                                                               |

Reviewer 2: ```suggestion
| --scope-datacenter    | Search in the following datacenter(s) (can be a regexp).                                                                                                                                                                               |
```

> | --scope-cluster       | Search in following cluster(s) (can be a regexp).                                                                                                                                                                                  |

Reviewer 2: ```suggestion
| --scope-cluster       | Search in the following cluster(s) (can be a regexp).                                                                                                                                                                                  |
```

> | --scope-host          | Search in following host(s) (can be a regexp).                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| --scope-host          | Search in the following host(s) (can be a regexp).                                                                                                                                                                                     |
```

> | --display-description | Display virtual machine description.                                                                                                                                                                                               |

Reviewer 2: ```suggestion
| --display-description | Display the description of the virtual machine.                                                                                                                                                                                               |
```

| --warning-*           | Warning threshold. Can be: 'swap-in', 'swap-out'.                                                                                                                                                                                  |
| --critical-*          | Critical threshold. Can be: 'swap-in', 'swap-out'.                                                                                                                                                                                 |

</TabItem>
<TabItem value="Vm-Thinprovisioning" label="Vm-Thinprovisioning">

| Option                    | Description                                                                                  |
|:--------------------------|:---------------------------------------------------------------------------------------------|
| --vm-hostname             | VM hostname to check. If not set, we check all VMs.                                          |

> | --filter                  | VM hostname is a regexp.                                                                     |

Reviewer 2: ?

| --filter-description      | Filter also virtual machines description (can be a regexp).                                  |
| --filter-os               | Filter also virtual machines OS name (can be a regexp).                                      |

> | --scope-datacenter        | Search in following datacenter(s) (can be a regexp).                                         |

Reviewer 2: ```suggestion
| --scope-datacenter        | Search in the following datacenter(s) (can be a regexp).                                         |
```

> | --scope-cluster           | Search in following cluster(s) (can be a regexp).                                            |

Reviewer 2: ```suggestion
| --scope-cluster           | Search in the following cluster(s) (can be a regexp).                                            |
```

> | --scope-host              | Search in following host(s) (can be a regexp).                                               |

Reviewer 2: ```suggestion
| --scope-host              | Search in the following host(s) (can be a regexp).                                               |
```

| --disconnect-status       | Status if VM disconnected (default: 'unknown').                                              |

> | --nopoweredon-skip        | Skip check if VM is not poweredOn.                                                           |

Reviewer 2: ```suggestion
| --nopoweredon-skip        | Skip check if VM is not powered on.                                                           |
```

> | --display-description     | Display virtual machine description.                                                         |

Reviewer 2: ```suggestion
| --display-description     | Display the description of the virtual machine.                                                         |
```

> | --thinprovisioning-status | Thinprovisioning status (default: none) Example: 'active,CRITICAL' or 'notactive,WARNING'    |

Reviewer 2: ```suggestion
| --thinprovisioning-status | Thin provisioning status (default: none). Example: 'active,CRITICAL' or 'notactive,WARNING'    |
```

</TabItem>
<TabItem value="Vm-Tools" label="Vm-Tools">

| Option                      | Description                                                   |
|:----------------------------|:--------------------------------------------------------------|
| --vm-hostname               | VM hostname to check. If not set, we check all VMs.           |

> | --filter                    | VM hostname is a regexp.                                      |

Reviewer 2: ?

| --filter-description        | Filter also virtual machines description (can be a regexp).   |
| --filter-os                 | Filter also virtual machines OS name (can be a regexp).       |

> | --scope-datacenter          | Search in following datacenter(s) (can be a regexp).          |

Reviewer 2: ```suggestion
| --scope-datacenter          | Search in the following datacenter(s) (can be a regexp).          |
```

> | --scope-cluster             | Search in following cluster(s) (can be a regexp).             |

Reviewer 2: ```suggestion
| --scope-cluster             | Search in the following cluster(s) (can be a regexp).             |
```

> | --scope-host                | Search in following host(s) (can be a regexp).                |

Reviewer 2: ```suggestion
| --scope-host                | Search in the following host(s) (can be a regexp).                |
```

| --disconnect-status         | Status if VM disconnected (default: 'unknown').               |

> | --nopoweredon-skip          | Skip check if VM is not poweredOn.                            |

Reviewer 2: ```suggestion
| --nopoweredon-skip          | Skip check if VM is not powered on.                            |
```

> | --empty-continue            | Ask to the connector that an empty response is ok.            |

Reviewer 2: ?

> | --display-description       | Display virtual machine description.                          |

Reviewer 2: ```suggestion
| --display-description       | Display the description of the virtual machine.                          |
```

| --tools-notinstalled-status | Status if vmtools is not installed (default: critical).       |
| --tools-notrunning-status   | Status if vmtools is not running (default: critical).         |

> | --tools-notup2date-status   | Status if vmtools is not upd2date (default: warning).         |

Reviewer 2: ```suggestion
| --tools-notup2date-status   | Status if vmtools is not up to date (default: warning).         |
```
